src/utils.ts

```diff
@@ -21,6 +21,24 @@ export const mac = ios || /Mac/.test(platform);
 * Utility functions used in one or more files
 */
 
+/**
+ * @internal
+ * @returns a number, representing ReturnType<typeof requestAnimationFrame>
+ * Expose a scheduler for after react's render cycle is over. Some
+ * internal functions use it, and testing infrastructure may use it as well
+ * see stackoverflow.com/questions/26556436/react-after-render-code/28748160#28748160
+ * 
+ * - If the owner argument is passed, an inner timeout of 0 is set as 'pendingTimeout'
+ *   so that it can be cancelled by the owner later
+ * - If an extraDelay is passed, the inner timeout waits Xms after the render cycle
+ */
+
+export function afterDOMUpdate(f: ()=>void, owner?:$TSFixMe, extraDelay?:number) {
+  return window.requestAnimationFrame(() => {
+    const timeout = setTimeout(f, extraDelay);
+    if(owner) owner.pendingTimeout = timeout;
+  });
+}
```

Comment:
I still don't like this. The contract between `afterDOMUpdate` and the owner isn't clear. I'd suggest using the same pattern that setTimeout and requestAnimationFrame use instead where you have one function that starts the asynchronous process, returning a handle, and a second function that takes a handle and cancels the asynchronous process for it. i.e. `afterDOMUpdate()` and `cancelAfterDOMUpdate`. If you want to be more object oriented, you can make the cancel function part of the handle that gets returned. Something like this:

```ts
export function afterDOMUpdate(f: ()=>void, extraDelay=0) {
  let timeoutHandle: ReturnType<typeof setTimeout>;
  const rafHandle = window.requestAnimationFrame(() => {
    timeoutHandle = setTimeout(f, extraDelay);
  });
  return {
    cancel: () => {
      window.cancelAnimationFrame(rafHandle);
      if (timeoutHandle) {
        clearTimeout(timeoutHandle);
      }
    }
  };
}
```

Then it would be used in a component like:

```ts
componentDidMount() {
  this.pendingTask = afterDOMUpdate(doSomethingLater);
}
componentWillUnmount() {
  this.pendingTask.cancel();
}
```

Now `afterDOMUpdate` doesn't need to know anything about the source from which it was called, and the source from which it was called doesn't need to conform to some interface that is specific to `afterDOMUpdate`.